homeassistant/components/melcloud/climate.py

```diff
@@ -32,39 +45,71 @@
     """Set up MelCloud device climate based on config_entry."""
     mel_devices = hass.data[DOMAIN][entry.entry_id]
     async_add_entities(
-        [AtaDeviceClimate(mel_device) for mel_device in mel_devices[DEVICE_TYPE_ATA]],
+        [
+            AtaDeviceClimate(mel_device, mel_device.device)
+            for mel_device in mel_devices[DEVICE_TYPE_ATA]
+        ]
+        + [
+            AtwDeviceZoneClimate(mel_device, mel_device.device, zone)
+            for mel_device in mel_devices[DEVICE_TYPE_ATW]
+            for zone in mel_device.device.zones
+        ],
         True,
     )
 
 
-class AtaDeviceClimate(ClimateDevice):
-    """Air-to-Air climate device."""
+class MelCloudClimate(ClimateDevice, ABC):
```

Comment:
Do we need to inherit from `ABC`? We don't seem to define any abstract methods.

Reply:
This is just me absent-mindedly trying to make a linter happy. Removed.